src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -120,6 +120,14 @@ module sonic-device_metadata {
                     }
                     default enable;
                 }
+
```

Comment:
Are there any restrictions on the values cloudtype or region can have? What are the background checks done and if possible can they be incorporated in YANG model?

Reply:
There are no restriction on the values as far as we know.